service/raft/group0_state_machine.cc

```diff
@@ -348,7 +348,7 @@ future<> group0_state_machine::transfer_snapshot(raft::server_id from_id, raft::
 
     co_await _sp.mutate_locally({std::move(history_mut)}, nullptr);
   } catch (const abort_requested_exception&) {
-    throw raft::request_aborted();
+      throw raft::request_aborted(format("Abort requested while transferring snapshot from ID/IP: {}/{}, snapshot descriptor id: {}",  snp.idx, from_ip, snp.id));
```

Comment:
nit: added whitespace, which makes it inconsistent with the code around it.

I understand why you did it -- the code currently doesn't follow our style established elsewhere (using indentation of 2 spaces instead of 4). But if you decide to fix this, you should fix the entire function; and then this should be done in a separate commit dedicated to whitespace fix, so not to pollute this commit which changes the logic making it trickier to review.